ci/travis/determine_tests_to_run.py

```diff
@@ -55,7 +55,7 @@ def get_commit_range():
         base = event["pull_request"]["base"]["sha"]
         commit_range = "{}...{}".format(base, event.get("after", ""))
     elif os.environ.get("BUILDKITE"):
-        commit_range = "{}...{}".format(
+        commit_range = "origin/{}...{}".format(
```

Comment:
this is a bug in our CI, without `origin` prefix it only works with master base branch.

Reply:
cc @simon-mo can you check if this fix won't impact other parts?

Reply:
I guess there isn't no harm? In the CI environment all branches on from origin

Reply:
ah yes. this helps me to stack PRs: in this PR we try to merge into a test-wheels branch instead of master.
this fix helps buildkite to run 
git diff **origin**/test-wheels/introduce-allocator...{revision}
instead of 
git diff test-wheels/introduce-allocator...{revision}
in buildkite,

where the later fails.